### PR TITLE
Translated the main page to english.

### PR DESCRIPTION
**Pull request by fccoelho**
Now it will be the official language of the site.

### CODE REVIEW

**Comment by fccoelho** [comment]
@flavioamieiro could you review this?

File: templates/pages/indextimeline.html [hunk 1]
```diff
@@ -41,10 +41,9 @@
     <div class="container">
         <h1>InfoZika</h1>
 
-        <p>O projeto InfoZika Busca monitorar tudo o que se sabe sobre o vírus Zika e suas epidemias. Em breve
-            incluiremos informações similares sobre outras
-            <a href="https://pt.wikipedia.org/wiki/Arbov%C3%ADrus"> arboviroses</a>.</p>
-        <p> Fonte: <a href="http://www.ncbi.nlm.nih.gov/pubmed">PubMed</a></p>
+        <p>The infoZika project monitors everything that is known about Zika virus and Zika fever. We do this by analyzing the scientific publications about it.
```
Comment (flavioamieiro) [inline templates/pages/indextimeline.html:44]: This line is missing a `</p>` to close the paragraph.

**Comment by flavioamieiro** [comment]
Other than the missing close paragraph tag, I think this is perfect. I can fix that, merge and deploy if you want.

**Comment by fccoelho** [comment]
please do that.

On Wed, May 18, 2016 at 4:23 PM, Flávio Amieiro notifications@github.com
wrote:

> Other than the missing close paragraph tag, I think this is perfect. I can
> fix that, merge and deploy if you want.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/elisamussumeci/zika_virus_timeline/pull/25#issuecomment-220131386

## 

# Flávio Codeço Coelho

+55(21) 3799-5551
Professor
Escola de Matemática Aplicada
Fundação Getulio Vargas
Praia de Botafogo, 190 sala 312
Rio de Janeiro - RJ
22250-900
Brasil

**Pull request merged**